web_app/src/components/trips/trip-list.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {useEffect} from 'react'
+import TripSummary from './trip-summary'
+import {Link} from 'react-router-dom'
+import {useStoreActions, useStoreState} from "../../hooks"
+
+export default function TripList() {
+  const {items, error, loading} = useStoreState(state => state.trips)
+  const {fetch} = useStoreActions(actions => actions.trips)
+
+  useEffect(() => {
+    fetch()
```

Comment:
`fetch()` returnerer `() => unsubscribe()`, så hvis vi returnerer den funksjonen her, vil `unsubscribe` bli kalt når komponenten unmounter

```suggestion
    return fetch()
```